lib/builder.rb

```diff
@@ -418,7 +422,22 @@ def create_topic(user, action, data)
   end
 
   def send_message(user, action, data)
-    title = data[action['title']]
+
+    empty_data = false
+
+    data.each do |k, v|
+      if v.empty?
+        empty_data = true
+      end
+    end
+    # leave if there are any empty fields
+    return if empty_data
+
+    unless action['custom_title_enabled']
```

Comment:
@merefield The precedent to follow here is the custom_title conditionals at the start of create_topic and open_composer actions, the main difference is adding in wizard field interpolation:
```
    if action['custom_title_enabled']
      title = CustomWizard::Builder.fill_placeholders(action['custom_title'], user, data)
    else
      title = data[action['title']]
    end
```

Reply:
done pending commit